wire-library/wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -83,6 +83,11 @@ class SwiftGenerator private constructor(
     get() = schema.getType(this) is EnumType
   private val Type.typeName
     get() = type.typeName
+  private val ProtoType.isStringEncoded
```

Comment:
What about `SFIXED64` and `UINT64`?

Reply:
Hah I do not remember where I got this list from

Reply:
Or why it's different from `fun TypeName.needsStringEncoded()` which is _only_ Int types...
back to the docs

Reply:
OK unified and fixed up